docs/index.md

```diff
@@ -122,9 +122,9 @@ These are materials that cover advanced concepts and mechanisms for developing a
 
         :material-compass:{.nextflow-primary} Collections of Side Questions grouped around a particular theme or use case.
 
-    Training Collections combine multiple Side Quests in order to provide a comprehensive learning experience around a particular around a common theme or use case.
+    Training Collections combine multiple Side Quests in order to provide a comprehensive learning experience around a common theme or use case.
```

Comment:
tiny nitpick: we actually want 'particular' here, not 'common'

Reply:
Ok